src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -534,6 +535,62 @@ public string StopRecordingScreen(IScreenRecordingOptions options)
 
         #endregion Recording Screen
 
+        #region Compare Images
+
+        public FeaturesMatchingResult MatchImageFeatures(string base64Image1, string base64Image2, FeaturesMatchingOptions options = null)
+        {
+            var parameters = new Dictionary<string, object> {
+                { "mode", ComparisonMode.MatchFeatures },
+                { "firstImage", base64Image1 },
+                { "secondImage", base64Image2 }
+            };
+
+            if (options != null)
+            {
+                parameters.Add("options", options.GetParameters());
+            }
+
+            var result = Execute(AppiumDriverCommand.CompareImages, parameters);
+            return new FeaturesMatchingResult(result.Value as Dictionary<string, object>);
+        }
+
+        public OccurenceMatchingResult FindImageOccurence(string base64Image1, string base64Image2, OccurenceMatchingOptions options = null)
+        {
+            var parameters = new Dictionary<string, object> {
+                { "mode", ComparisonMode.MatchTemplate },
+                { "firstImage", base64Image1 },
+                { "secondImage", base64Image2 }
+            };
+
+            if (options != null)
```

Comment:
can we use [nullable](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/index) types notation?

Reply:
same comment to other methods

Reply:
I don't think so:
> T cannot be a reference type.
( from https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/index)

Reply:
@mykola-mokhnach Did you mean:
`options?.parameters.Add("options", options.GetParameters());`

Reply:
@akinsolb Yes, something like that to explicitly show that options is nullable